packages/Avatar/src/helpers.js

```diff
@@ -25,3 +25,8 @@ export function getAvatarColors(text) {
     backgroundColor: COLORS[index].background,
   };
 }
+
+export function getInitialsFromText(text, length = 1) {
```

Comment:
Oh, I was thinking it would go in the `@paprika/helpers` package, but this works too.  But maybe it would be nice to export it in the `/src.index.js` file as well, so consumers can get it like: 

```js 
import Avatar, { getInitialsFromText } from '@paprika/avatar';
```

instead of 

```js
import Avatar from '@paprika/avatar';
import { getInitialsFromText } from '@paprika/avatar/lib/helpers';
```

Reply:
I've updated the example story to grab the helper function as you describe

Reply:
Oh! I didn't realize it was already being exported from `index.js` 🤦 
https://github.com/acl-services/paprika/blob/master/packages/Avatar/src/index.js#L2